src/components/Header.tsx

```diff
@@ -0,0 +1,33 @@
+type Props = {
```

Comment:
```suggestion
interface NavigationHeaderProps = {
```

типы объектов объявляем через interface. type используется только в сложных кейсах типизирования

Reply:
done